app/src/main/java/uw/hcrlab/kubi/lesson/prompts/NamePrompt.java

```diff
@@ -70,6 +110,17 @@ public void afterTextChanged(Editable s) {
     }
 
     public void handleResults(Result res) {
+        NameResult result = (NameResult) res;
 
+        if(result.hasBlame()) {
+            Toast toast = Toast.makeText(getActivity().getApplicationContext(), result.getBlame(), Toast.LENGTH_SHORT);
+            toast.show();
+        }
```

Comment:
IDK honestly. Probably a placeholder, but I haven't had time to try other display methods. I think it would probably be good to display in the thought bubble with a red border instead of the current blue.